Makefile

```diff
@@ -18,6 +18,10 @@ OUT_DIR ?= _output
 OUT_BINPATH := $(OUT_DIR)/bin
 
 IMAGE_REPO ?= kubeedge
+
+PLATFORMS ?= "linux/amd64,linux/arm64"
```

Comment:
Can you add a description about full list of platforms?

Reply:
OK